packages/shared/src/components/sidebar/sections/BookmarkSection.tsx

```diff
@@ -1,51 +1,89 @@
 import React, { ReactElement } from 'react';
 import { ListIcon, SidebarMenuItem } from '../common';
-import { BookmarkIcon, PlusIcon } from '../../icons';
+import { ArrowIcon, BookmarkIcon, PlusIcon } from '../../icons';
 import { Section } from '../Section';
 import { webappUrl } from '../../../lib/constants';
 import { SidebarSettingsFlags } from '../../../graphql/settings';
 import { SidebarSectionProps } from './common';
 import { useLazyModal } from '../../../hooks/useLazyModal';
 import { LazyModal } from '../../modals/common/types';
+import { BookmarkReminderIcon } from '../../icons/Bookmark/Reminder';
+import {
+  useBookmarkFolderList,
+  useCreateBookmarkFolder,
+} from '../../../hooks/bookmark';
+import { useViewSize, ViewSize } from '../../../hooks';
+import { FolderIcon } from '../../icons/Folder';
 
 export const BookmarkSection = ({
   isItemsButton,
   ...defaultRenderSectionProps
 }: SidebarSectionProps): ReactElement => {
-  const { openModal } = useLazyModal();
+  const { openModal, closeModal } = useLazyModal();
+  const { folders } = useBookmarkFolderList();
+  const { createFolder } = useCreateBookmarkFolder();
+
+  const isLaptop = useViewSize(ViewSize.Laptop);
+  const rightIcon = !isLaptop && (() => <ArrowIcon className="rotate-90" />);
+
+  const onAddFolderClick = (
+    event: React.MouseEvent<HTMLAnchorElement | HTMLButtonElement>,
+  ) => {
+    event.preventDefault();
+    openModal({
+      type: LazyModal.BookmarkFolder,
+      props: {
+        onSubmit: async (folder) => {
+          await createFolder(folder);
+          closeModal();
+        },
+        folderCount: folders.length ?? 0,
+      },
+    });
+  };
 
   const menuItems: SidebarMenuItem[] = [
     {
       icon: (active: boolean) => (
         <ListIcon Icon={() => <BookmarkIcon secondary={active} />} />
       ),
-      title: 'Bookmarks',
+      title: 'Quick saves',
       path: `${webappUrl}bookmarks`,
       isForcedLink: true,
       requiresLogin: true,
+      rightIcon,
+    },
+    {
+      icon: (active: boolean) => (
+        <ListIcon Icon={() => <BookmarkReminderIcon secondary={active} />} />
+      ),
+      title: 'Read it later',
+      path: `${webappUrl}bookmarks/read-it-later`,
```

Comment:
Similar to Slack, I think we can call it `later` at times.
```suggestion
      path: `${webappUrl}bookmarks/later`,
```